### PR TITLE
Fleshing out case details page

### PR DESCRIPTION
**Pull request by AriHrannar**
## Status
**Ready**

## Description
Adding some basic information to the case detail page and styling it a bit. Not the best thing ever but it displays a different message / color depending on the status. More information should be added in further enhancements

## Example | Screenshots
selection_002

### CODE REVIEW

File: src/Components/Cases/constants.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  Unknown: 'Woops, we are not sure what is happening with this case',
+  'In progress': 'Dont worry, this case is currently being worked on',
+  Rejected: 'This case was rejected!',
+  Answered: 'This request has been answered',
+  'Has not been answered': 'This request has not been answered',
+  Withdrawn: 'This case was withdrawn',
+  Passed: 'Nice! This was passed!',
+};
+
+export const red = 'rgb(245,14,30)';
```
Comment (Ragnar-H) [inline src/Components/Cases/constants.js]: It's usually not a good idea to use colour names in colour constants.

Instead they should represent the semantic use:

`export const warning = 'rgb(245, 14, 30)'`

This is good for two reasons:

1. If we decide some day that warnings should be orange; We won't end up with an orange warning label with the style 'red'
2. It's immediately clear which colour to use from the style guide; Is this a serious alert I'm showing the user? Then I'll use the `warning` constant

Pretty nitpicky comments I know. But it's something I've struggled with before :P

File: src/Components/Cases/constants.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+export const STATUS_TO_DESCRIPTION = {
+  Unknown: 'Woops, we are not sure what is happening with this case',
+  'In progress': 'Dont worry, this case is currently being worked on',
+  Rejected: 'This case was rejected!',
+  Answered: 'This request has been answered',
+  'Has not been answered': 'This request has not been answered',
+  Withdrawn: 'This case was withdrawn',
+  Passed: 'Nice! This was passed!',
```
[the remark this answers is not in the document]
Comment (AriHrannar) [reply src/Components/Cases/constants.js]: That is very true!

File: src/Components/Cases/constants.js (hunk 1 above)
Comment (AriHrannar) [reply src/Components/Cases/constants.js]: thats true, I have changed this

Comment (AriHrannar) [reply src/Components/Cases/constants.js]: means we can also reuse this colors across the board later (and maybe abstract them into another file if needed)

**Pull request merged**